moped-database/migrations/1695084733428_more_work_activity_fields/up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE moped_proj_work_activity RENAME COLUMN interim_work_order_id_old TO interim_work_activity_id;
+
+ALTER TABLE moped_proj_work_activity
+    ADD COLUMN work_order_url text,
+    ADD COLUMN reference_id text 
+        GENERATED ALWAYS AS (COALESCE(interim_work_activity_id, 'MPD-' || project_id::text || '-' || id::text)) STORED;
+
+COMMENT ON COLUMN moped_proj_work_activity.work_order_url IS 'External link to a related work order. E.g., to the Knack Data Tracker';
+COMMENT ON COLUMN moped_proj_work_activity.reference_id IS 'An auto-generated ID that can be submitted to work order systems to indicate that the work originated from a Moped project.';
```

Comment:
There is a convention in the Signs & Markings work order system where those who submit request work orders use a "Requester ID" to refer back to their own tracking system.

Moped is going to become the tracking system for a lot of signs and markings work orders, so the idea is that we will generate the "requester_id" in the form at of `MPD-<project_id>-<activity_id>`. E.g. `MPD-2340-1023`. However, if the work activity was migrated from access, we will show the original work activity ID in Moped, since that ID already ties back to the Signs & Markings app.

For example, an old reference ID would be `ATSD-1868`, as pictured here:
<img width="1314" alt="Screenshot 2023-10-06 at 2 02 24 PM" src="https://github.com/cityofaustin/atd-moped/assets/14793120/607015f6-fe18-410e-aabb-f0516937a1c4">